colorcoder/Color.java

```diff
@@ -0,0 +1,10 @@
+package colorcoder;
```

Comment:
Who depends on interface Color and its significance

Reply:
As discussed on the session, i will try to find out any other way to achieve this.